### DISCUSSION TITLE
December/January sprint?

### DISCUSSION THREAD

**Discussion by StanczakDominik** [PlasmaPy/PlasmaPy#1349] [General]
During [last telecon](https://hackmd.io/@plasmapy/ByS2k2pKF), I think it was @namurphy who floated the idea of having a 3-4 hour dev sprint via zoom. @rocco8773 mentioned wanting to wait for January.

But I still don't see why we can't have two of those! (if this is another case of me focusing on the problem and forgetting the human, please say so @rocco8773, I don't want to hurt any feelings here).

So... who's in? And when do we schedule that?

**Comment by namurphy** [comment]
I'd be happy to try this out!  Doing this once in each of December & January would work for me.  We could perhaps talk about the times next Tuesday, but I don't plan on traveling over the next month or two so my schedule should be pretty flexible.  It'd be helpful to choose the topic(s) of the sprint in advance.  I'll bring snacks! 😹

**Comment by rocco8773** [comment]
I'm more that happy for there to be a sprint before January.  I just don't think I'll have the time to participate.  By all means, proceed!  I would love to see something like this to happen once or twice a month going forward!
